plugins/SitesManager/templates/siteWithoutData.twig

```diff
@@ -12,6 +12,8 @@
 
     <script type="text/javascript" charset="utf-8">
         $(document).ready(function () {
+            $('body').addClass('white-background')
+            $('.top_controls').addClass('white-background')
```

Comment:
Might it be smarter to add something like `{% set bodyId = 'site-without-data' %}` to the template, so you can easily define the styles in css using the body id instead of heaving to set the white background class using javascript?

Reply:
Thanks, I didn't knew this was possible too :+1: 
Updated